### ISSUE TITLE
click test helper does not pass the correct params to mousedown, mouseup

### ISSUE THREAD

**Issue by BryanCrotaz** [emberjs/ember.js#14101]
mousedown and mouseup events have a button param:

```
... my component ...
	mouseDown: function(event) {
		if (event.button !== 0) {
			// not left button
			return false;
		}
                ...
        }
```
click test helper is:
```
  function click(app, selector, context) {
    var $el = app.testHelpers.findWithAssert(selector, context);
    _emberMetalRun_loop.default($el, 'mousedown');

    focus($el);

    _emberMetalRun_loop.default($el, 'mouseup');
    _emberMetalRun_loop.default($el, 'click');

    return app.testHelpers.wait();
  }
```

It should pass left mouse button:
```
    _emberMetalRun_loop.default($el, 'mousedown', {button:0} );
    focus($el);
    _emberMetalRun_loop.default($el, 'mouseup', {button:0} );
```

I can make a PR if this is acceptable?

Thinking of adding a `rightclick` helper at the same time

**Comment by pixelhandler** [comment]
@BryanCrotaz yeah it's worth making a PR to pass the button property. I'm curious, are there other properties of the event that are missing? https://developer.mozilla.org/en-US/docs/Web/Events/mouseup

Also it would be a good idea to start with a failing test for the PR.

**Comment by rwjblue** [comment]
@BryanCrotaz - Ping?

**Comment by Serabe** [comment]
@BryanCrotaz - hello! The PR would be really interesting. Do you have some time for it? If you need some help with the process, do not hesitate on letting us now in the Slack community and we will help you. If you are not registered, use the [Slackin app](https://ember-community-slackin.herokuapp.com/

Thank you for your contribution!

**Comment by BryanCrotaz** [comment]
PR #14274

**Comment by BryanCrotaz** [comment]
added tests and docs

**Comment by BryanCrotaz** [comment]
Thoughts?

**Comment by pixelhandler** [comment]
@BryanCrotaz @rwjblue is this still an issue, perhaps we should close or create a new reproduction of this, what do you think? My bet is that this is no longer an issue using ember-test-helpers https://github.com/emberjs/ember-test-helpers/blob/master/API.md

**Comment by BryanCrotaz** [comment]
This is still an issue
